backend/asset/s3_videos.go

```diff
@@ -0,0 +1,70 @@
+package asset
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/ansel1/merry/v2"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+)
+
+func getSignedAudiolessVideoURLForAssetID(context context.Context, services externalServices, config config, assetID int32) (*string, error) {
```

Comment:
Why `Audioless`? I.e. Do we care?

Reply:
Well just wanted it to be clear so you dont expect audio